core/src/test/java/io/questdb/test/griffin/wal/WalWriterFuzzTest.java

```diff
@@ -470,31 +482,13 @@ private void assertRandomIndexes(String tableNameNoWal, String tableNameWal, Rnd
         try (TableReader reader = newTableReader(configuration, tableNameNoWal)) {
             if (reader.size() > 0) {
                 TableReaderMetadata metadata = reader.getMetadata();
-                int columnIndex = rnd.nextInt(metadata.getColumnCount());
-                for (int i = 0; i < metadata.getColumnCount(); i++) {
-                    columnIndex += i;
+                for (int columnIndex = 0; columnIndex < metadata.getColumnCount(); columnIndex++) {
                     columnIndex = columnIndex % metadata.getColumnCount();
```

Comment:
nit: this line is now redundant as the loop has the `columnIndex < metadata.getColumnCount()` check.